### DISCUSSION TITLE
Restart postal-worker if rabbit connection is lost

### DISCUSSION THREAD

**Discussion by chrism417** [postalserver/postal#2099] [Feature suggestions]
When rabbit connection is lost, the requeuer will restart, however the worker does not.

Can this functionality the requeuer has be implemented into the worker?

**Comment by willpower232** [comment]
I think its more of an unintended side effect but if someone who knows Ruby on Rails has an idea about how this works, that would be good to know :sweat_smile: 

https://github.com/postalserver/postal/blob/main/lib/postal/message_requeuer.rb
